### DISCUSSION TITLE
Specifying a runner version in a GitHub hosted workflow

### DISCUSSION THREAD

**Discussion by jameshnsears** [actions/runner#2053] [General]
Hi - runner version '2.293.0' used to build my android app. However, '2.294.0' does not. 

I'm using a direct branch from the commit that worked against '2.293.0' - so nothing has changed my end.

This is all happening in a GitHub hosted 'macOS-latest' environment.

So, I'm wondering, is it possible to specify '2.293.0' in my GitHub hosted workflow?

PS - my problem might be related to a change in reactivecircus/android-emulator-runner@v2 that I'm using in my workflow file - but I thought I'd investigate changing the runner version first.

**Comment by henrygriffiths** [comment]
Hi @jameshnsears,

I'm not sure about specifying version on the GitHub hosted runner, but you can disable updates on self-hosted runners if that is an option for you: https://github.blog/changelog/2022-02-01-github-actions-self-hosted-runners-can-now-disable-automatic-updates/

**Reply by jameshnsears** [reply]
Hi @henrygriffiths - thanks for the reply, but, alas, this isn't an option for me.

If I'm going to use this runner, then it needs to be GitHub hosted.
